test/Constraints/pack_expansion_types.swift

```diff
@@ -300,3 +300,169 @@ func test_one_element_tuple_vs_non_tuple_matching() {
     test(V<Int>.self) // Ok
   }
 }
+
+// MARK: lValue'd packing (Issue 85924)
```

Comment:
We don't use MARK: in tests, how about just saying "// Ensure correct behavior of lvalues of tuples" or something